shared/constants/tabs.js

```diff
@@ -20,7 +20,7 @@ export const settingsTab = 'tabs:settingsTab'
 
 const prettyNames = {
   [startupTab]: null,
-  [folderTab]: 'Folder',
+  [folderTab]: 'Folders',
```

Comment:
This string is displayed in the app as the tab headline, so I think Folders is correct, not Folder.